### ISSUE TITLE
Pin aliases not working

### ISSUE THREAD

**Issue by danielmahon** [bryan-m-hughes/raspi-io#21]
Using the Raspi2 in the following example only the WiringPi pin numbers work. With `GPIO-18` and `P1-12` the `motionstart` event is not fired and no errors occur. I have tried with other pins as well.

```
  // Create a new `motion` hardware instance.
  var motion = new five.IR.Motion(1); // "GPIO-18" and "P1-12" doesn't work

  // "calibrated" occurs once, at the beginning of a session,
  motion.on("calibrated", function() {
    console.log("calibrated");
  });

  // "motionstart" events are fired when the "calibrated"
  // proximal area is disrupted, generally by some form of movement
  motion.on("motionstart", function() {
    console.log("motionstart");
    intruder = true;

    sendMessage('Motion Detected!');

  });
```

**Comment by bryan-m-hughes** [comment]
Which version of raspi-io are you using?

**Comment by danielmahon** [comment]
@bryan-m-hughes v2.2.2

**Comment by bryan-m-hughes** [comment]
I just published v2.2.3 which fixes this issue. Thanks for the bug report!
